packages/language-server/src/core/config/ConfigManager.ts

```diff
@@ -1,4 +1,4 @@
-import { merge } from 'lodash';
+import { merge, get } from 'lodash';
```

Comment:
i thought you had removed lodash previously?

Reply:
Not yet, it's still used here and inside the pluginHost. I am still planning to get it removed soonish though